tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -24,9 +25,13 @@
 @pytest.mark.instances(["c4.xlarge", "c5.xlarge"])
 @pytest.mark.schedulers(["sge"])
 @pytest.mark.usefixtures("region", "os", "instance")
-def test_ebs_single(scheduler, pcluster_config_reader, clusters_factory):
+def test_ebs_single(scheduler, pcluster_config_reader, clusters_factory, kms_key_factory):
     mount_dir = "ebs_mount_dir"
-    cluster_config = pcluster_config_reader(mount_dir=mount_dir)
+    kms_key_id = kms_key_factory.create_kms_key(scheduler)
+    ec2_iam_role_name = kms_key_factory.iam_role
```

Comment:
minor: it doesn't look like you need to define the `ec2_iam_role_name` variable. maybe you can delete this line and in the next line's function call use `ec2_iam_role=kms_key_factory.iam_role` ?

Reply:
Right! Done!